README.md

```diff
@@ -61,8 +61,45 @@ $ oc adm taint nodes <NodeNames> node.ocs.openshift.io/storage=true:NoSchedule
 
 ### Installation
 
+
 The OCS operator can be installed into an OpenShift cluster using Operator Lifecycle Manager (OLM).
 
+Option 1: Using `make install`  
```

Comment:
Can you give Option 1 & Option 2 as ####

Reply:
If I remember correctly, you didn't want Option 1 and Option 2 to appear in the table of contents, right?

Reply:
Hm don't think so, may be the discussion was about using bullets instead of number.
Please add #### to this.

Reply:
done